src/components/Header/Header.jsx

```diff
@@ -1,18 +1,72 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import styles from './Header.module.css';  
+import React, { useState } from "react";
+import styles from "./Header.module.css";
+import Logo from "../../assets/logo.svg";
+import Button from "../utility/Button/Button";
+import { Link } from "react-router-dom";
+import closeIcon from "../../assets/icon-close.svg";
+import hamburger from "../../assets/icon-hamburger.svg";
+import Typography from "../utility/Typography/Typography";
 
 const Header = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+
   return (
-    <header className={styles.header}>
-      <nav>
-        <ul>
-          <li><Link to="/">Home</Link></li>
-          <li><Link to="/about">About</Link></li>
-          <li><Link to="/contact">Contact Us</Link></li>
-        </ul>
-      </nav>
-    </header>
+    <>
+      <header className={styles.header}>
+        <div className={styles.navLinks}>
+          <img src={Logo} alt="logo" className={styles.logo} />
+          <Link
+            to="/"
+            className={`${styles.navLink} ${styles.nav}`}
+          >
+            <Typography elType="body4">home</Typography>
+          </Link>
+          <Link
+            to="/about"
+            className={`${styles.navLink} ${styles.nav}`}
+          >
+            <Typography elType="body4">about</Typography>
+          </Link>
+        </div>
+
+        <Button elType="primaryWhite" className={styles.nav}>
+          contact us
+        </Button>
```

Comment:
My big issue: This "button" should actually be a Link to the /contact page with styling to look like a button. Same for the copy in the mobile menu, and actually the same for the `ReadyToStart` "button" though in a different color. 

Do we make a component for this? Can we import the Button styles to apply directly to an `a` tag instead?

Or (too much work, not enough time, probably) do we develop a modal to bring up on clicking these "contact us" buttons? (I also dislike that from an accessibility perspective, but it's technically an option.)

Reply:
no worries, I will take a look on this today

Reply:
The button functionality looks great!